src/Microsoft.AspNetCore.Mvc.ViewFeatures/Internal/TempDataApplicationModelProvider.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using Microsoft.AspNetCore.Mvc.ApplicationModels;
+using Microsoft.Extensions.Internal;
+
+namespace Microsoft.AspNetCore.Mvc.ViewFeatures.Internal
+{
+    public class TempDataApplicationModelProvider : IApplicationModelProvider
+    {
+        /// <inheritdoc />
+        public int Order { get { return -1000 + 10; } }
```

Comment:
This value is the same as other ApplicationModelProviders like `AuthorizationApplicationModelProvider` and `CorsApplicationModelProvider`. Is that correct?